1-introduction/1.3.4-passwords-and-keys.md

```diff
@@ -57,7 +62,9 @@ cat id_rsa.pub >> ~/.ssh/authorized_keys
 rm id_rsa.pub
 ```
 
-SSH will automatically try to authenticate with the .ssh/id_rsa key so you should be able to log into the host without a password now.
+?> **Note:**  The ed25519 algorithm provides better security and faster performance than the default RSA, so is the preferred algorithm. But as of 10/2022 it is [still not supported by Azure](https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/ed25519-ssh-keys). If you are not interacting with Azure VMs, consider using ed25519 ( `ssh-keygen -t ed25519` then copy the file named `id_ed25519.pub`).
```

Comment:
I see you do talk about this, I just needed to keep reading. Boo on Azure for not supporting ed25519 still.

Bootcampers will be creating Azure VM's so I guess we cannot use ed25519 yet 😢